hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainerData.java

```diff
@@ -133,6 +135,11 @@ public String getMetadataPath() {
     return metadataPath;
   }
 
+  @VisibleForTesting
+  public File getContainerFile() {
```

Comment:
We don't need this method. We already have methods to get `metadataPath` and `containerID`, using which we can construct `ContainerFile` object.